docs/getting_started/job_monitoring.md

```diff
@@ -0,0 +1,149 @@
+# Managing Your Jobs
+
+Jobs can be managed via the (web dashboard | https://dashboard.conductortech.com)
+
+**SCREENSHOT OF CONDUCTOR DASHBOARD**

```

Comment:
is the image supposed to be here already or is the remaining placeholder intentional?

Reply:
Thanks for catching that. I did put the screenshot in but forgot to remove that.